e2e/commands/start/start_test.go

```diff
@@ -296,4 +296,18 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"repository_name\": \"%s\"`, repositoryName)))
 		})
 	})
+
+	When("--information-severity is passed", func() {
+		BeforeEach(func() {
+			flags = map[string]string{
+				testutil.StartFlagProjectPath:         testutil.JavaScriptExample1,
+				testutil.StartFlagInformationSeverity: "true",
+			}
+		})
+
+		It("Checks if the information severity property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"enable_information_severity\": true`))
+			Expect(session.Out.Contents()).NotTo(ContainSubstring("Horusec not show info vulnerabilities in this analysis"))
```

Comment:
Is this correct? This message should only appear when the flag has not been set.

`Horusec not show info vulnerabilities in this analysis`

Reply:
Yes, that's why I used `NotTo`

Reply:
I get it!